src/main/java/me/ryanhamshire/GriefPrevention/EntityEventHandler.java

```diff
@@ -379,23 +377,99 @@ public void onEntityInteract(EntityInteractEvent event)
     @EventHandler(ignoreCancelled = true, priority = EventPriority.LOWEST)
     public void onEntityExplode(EntityExplodeEvent explodeEvent)
     {
-        this.handleExplosion(explodeEvent.getLocation(), explodeEvent.getEntity(), explodeEvent.blockList());
+        // If there aren't any affected blocks, there's nothing to do. Vanilla mob griefing rule causes this.
```

Comment:
diff makes it look like you just undid a helper method and have duplicate code, I'll revisit this logic later.

Reply:
ah, you added that on to this PR. Yea if I got to reviewing this a couple days ago I guess it would've been a little easier to review lol.

Reply:
Yeah, sadly EntityExplodeEvent and BlockExplodeEvent don't share a superclass (which kinda makes some sense, but is annoying) so it's minimal duplicate code in their listeners for determining which handler to pass off to or make one superhandler that is a mess. I suppose we could also build a wrapper like we do for damage handling.

Reply:
yea, for now this is ok, but ideally would like to move to some sort of funnel-like system later on (prolly a v18 or v19 thing) - which at least from what I recall, is ultimately a wrapper.